server/datasources.json

```diff
@@ -0,0 +1,6 @@
+{
+  "db": {
+    "name": "db",
```

Comment:
This is a workaround from Workspace@3... can we drop this now in favor of the new datasource...

Especially the fact that we are using the `memory` connector.